src/main/resources/org/jenkinsci/account/taglib/layout.jelly

```diff
@@ -27,244 +27,46 @@
 <!-- Non-obtrusive CSS styles -->
 <link href='https://www.jenkins.io/assets/bower/ionicons/css/ionicons.min.css' media='screen' rel='stylesheet'/>
 <link href='https://www.jenkins.io/css/footer.css' media='screen' rel='stylesheet'/>
-    <script src="/webjars/jquery/3.6.1/jquery.js"/>
-    <script src="/webjars/jquery-ui/1.13.2/jquery-ui.js"/>
 </head>
 <body style="min-height:100%; height:100%; padding-top:100px">
-    <script src="https://unpkg.com/@webcomponents/webcomponentsjs@2.6.0/webcomponents-loader.js"/>
-    <script src="https://unpkg.com/lit@2.4.0/polyfill-support.js"/>
-    <script data="jio" src="https://unpkg.com/@halkeye/jenkins-io-components?module" type="module"/>
-    <script data="jio" nomodule="" src="https://unpkg.com/@halkeye/jenkins-io-components"/>
-
-    <jio-navbar class="fixed-top" property="https://accounts.jenkins.io"/>
-
-    <div class='container pb-5' style="min-height:55%;">
-        <j:if test="${app.isLoggedIn() or it.isLoggedIn()}">
-            <nav class='navbar navbar-expand-md navbar-light' id="ji-toolbar">
-                <a class='navbar-brand' href='/'>
-                    My Account
-                </a>
-                <button class="navbar-toggler" type="button" data-toggle="collapse"
-                        data-target="#navbarSupportedContent"
-                        aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-                    <span class="navbar-toggler-icon"/>
-                </button>
-                <div class='collapse navbar-collapse' id='CollapsingNavbar2'>
-
-                    <ul class='navbar-nav mr-auto'>
-                        <j:if test="${it.isAdmin() or app.isAdmin()}">
-                            <li class="nav-item">
-                                <a href="/admin" class="active nav-link">Admin</a>
-                            </li>
-                        </j:if>
-                        <li class="nav-item">
-                            <a href="/myself" class="active nav-link">Profile</a>
-                        </li>
-
-                        <li class="nav-item">
-                            <a href="/logout" class="active nav-link">
-                                Logout
-                                <i class="fa fa-sign-out" aria-hidden="true"/>
-                            </a>
-                        </li>
-                    </ul>
-                </div>
-            </nav>
+  <jio-navbar class="fixed-top" property="https://accounts.jenkins.io">
+    <j:if test="${app.isLoggedIn() or it.isLoggedIn()}">
+      <div slot="rightMenuItems">
+        <j:if test="${it.isAdmin() or app.isAdmin()}">
+          <jio-navbar-link href="/admin">Admin</jio-navbar-link>
         </j:if>
+        <jio-navbar-link href="/myself">Profile</jio-navbar-link>
+        <jio-navbar-link href="/logout">Logout <ion-icon name="log-out-outline" /></jio-navbar-link>
```

Comment:
the ion-icon doesn't seem to be showing for me

![image](https://user-images.githubusercontent.com/21194782/196123132-52b50e6a-24bd-42e4-bf1d-97cccc4e551a.png)